src/rpc/fees.cpp

```diff
@@ -47,8 +47,11 @@ static RPCHelpMan estimatesmartfee()
                     {
                         {RPCResult::Type::STR, "", "error"},
                     }},
-                {RPCResult::Type::NUM, "blocks", "block number where estimate was found\n"
-                "The request target will be clamped between 2 and the highest target\n"
+                {RPCResult::Type::NUM, "blocks", "The returned fee rate estimate will likely make transaction begin\n"
+                "confirming confirm within this value. If an estimate for the exact\n"
+                "conf_target cannot be provided, the closest possible target\n"
+                "for which an estimate can be provided will be returned.\n"
+                "The requested conf_target will be clamped between 2 and the highest target\n"
```

Comment:
Might it make more sense to change this to something simpler:

```
The number of blocks from the current tip where the estimate was found.
```

And amend the doc on L38 about the confirmation target range from `1` to `2` to avoid having to explain it in `blocks`:

```
            {"conf_target", RPCArg::Type::NUM, RPCArg::Optional::NO, "Confirmation target in blocks (2 - 1008)"},
```

The error could be moved to the error description string:

```
                {RPCResult::Type::ARR, "errors", /*optional=*/true, "Errors encountered during processing. An error is returned if not enough transactions and blocks have been observed to make an estimate for any number of blocks."
```